rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -60,6 +60,12 @@ public enum BridgeEvents {
                     new CallTransaction.Param(false, "activationHeight", SolidityType.getType(SolidityType.INT256))
             }
     ),
+    COMMIT_FEDERATION_FAILED("commit_federation_failed",
+        new CallTransaction.Param[]{
+            new CallTransaction.Param(false, "proposedFederationRedeemScriptSerialized", SolidityType.getType(SolidityType.BYTES)),
+            new CallTransaction.Param(false, "blockNumber", SolidityType.getType(SolidityType.INT256))
```

Comment:
Why do you need to include this value? You can get it from the transaction where the event is included. Maybe you would like to store the block where the federation was proposed or something else, but I don't think this adds any value.